homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -69,7 +80,41 @@
     vol.Optional(ATTR_RC_DURATION): cv.positive_int,
 })
 
+SERVICE_SCHEMA_CLEAN_ZONE = VACUUM_SERVICE_SCHEMA.extend({
```

Comment:
This one should be removed, just `clean_zones` should be able to handle the cleaning of 1 zone.